src/transformers/integrations.py

```diff
@@ -1299,6 +1305,114 @@ def on_train_end(self, args, state, control, **kwargs):
             self.tracker.stop()
 
 
+class ClearMLCallback(TrainerCallback):
+    """
+    A [`TrainerCallback`] that sends the logs to [ClearML](https://clear.ml/).
+
+    Environment:
+            CLEARML_PROJECT (`str`, *optional*, defaults to "HuggingFace Transformers"):
+                ClearML project name. If not set deafault project name will be assigned.
+            CLEARML_TASK (`str`, *optional* defaults to `"Trainer"`):
+                ClearML task name. If not set default task name will be assigned.
+    """
+
+    def __init__(self):
+        has_clearml = is_clearml_available()
+
+        if has_clearml:
```

Comment:
```suggestion
        if is_clearml_available():
```
I don't think you need to store this result ;-)